build-system/server-inabox-template.html

```diff
@@ -0,0 +1,98 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>In-a-box Envelope</title>
+  <link rel="canonical" href="http://nonblocking.io/" >
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style>
+    body {
+      margin: 0;
+    }
+    iframe {
+      border: 1px solid gray;
+    }
+  </style>
+</head>
+<body>
+  <h1>In-a-box Envelope</h1>
+  <div>url: AD_URL</div>
+  <div>size: AD_WIDTHxAD_HEIGHT</div>
+
+  <div style="background: #eee; overflow: hidden;; height: OFFSET;">scroll down to see the ad</div>
+
+  <iframe
+      name="inabox-atf"
+      src="AD_URL"
+      width="AD_WIDTH" height="AD_HEIGHT"
+      scrolling="no"
+      marginwidth="0"
+      marginheight="0"
+      frameborder="0">
+  </iframe>
+
+  <script>
```

Comment:
Can we do this in our `server.js`? I'd definitely prefer if we did that... Or do you mean we use a `<script>` for this?